extensions/BMO/template/en/default/account/create.html.tmpl

```diff
@@ -94,6 +94,14 @@ function onSubmit() {
   %]
   </table>
 
+  <h2>I want to try out [% terms.Bugzilla %]</h2>
```

Comment:
<h2>"I'm here for my class"</h2>
<div id="left-blurb">
<p><strong>Inappropriate use of this service will result in your account being banned and your institution blocked.</strong></p>
<p>If you have been told to create bugs as part of a school assignment, go to <a href="https://bugzilla-dev.allizom.org">bugzilla-dev.allizom.org</a>, and log in using your GitHub account.<p>
<p>You can also set up your own instance of Bugzilla for evaluation or classroom training by<br>following the directions on <a href="https://github.com/mozilla-bteam/bmo/">our project repository</a>.</p>
</div>

Reply:
I think the header and message proposed by @dklawren are okay since not everyone is a student. Instead of replacing “I want to try out Bugzilla,” maybe we could just add the “I’m here for my class” section separately?

Reply:
Also we need to make sure the message only appears on [production](https://bugzilla.mozilla.org/createaccount.cgi), otherwise the user flow could be an infinite loop 😼

Reply:
I agree with @kyoshino that the message shouyld be worded in such a way to include the class members but also others who are there just to try out Bugzilla for personal interests. 

Maybe 

<h2>Inappropriate uses of Bugzilla</h2>
<div id="left-blurb">
<p><strong>Inappropriate use of this service will result in your account being banned and your institution blocked.</strong></p>
<ul>
<li>Creating bugs as part of a school assignment.</li>
<li>Trying out Bugzilla to see if it suits your needs or testing out some part of Bugzilla for proper functionality.</li>
</ul>
<p>Instead, go to <a href="https://bugzilla-dev.allizom.org">bugzilla-dev.allizom.org</a>, and log in using a GitHub account.</p>
<p>Note: You can also set up your own instance of Bugzilla for evaluation or classroom training by following the directions on <a href="https://github.com/mozilla-bteam/bmo/">our project repository</a>.</p>
</div>

Reply:
I like this one! :shipit: